### PR TITLE
Update DiscordSRV, Created listeners

### PR DESCRIPTION
**Pull request by Vankka**
Make descriptions much more specific (changed Guild -> Discord server)

### CODE REVIEW

**Comment by Vankka** [comment]
Closes https://github.com/plan-player-analytics/Plan/issues/1221

File: src/main/java/com/djrapitops/extension/DiscordSRVExtensionFactory.java [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
 package com.djrapitops.extension;
 
+import com.djrapitops.plan.extension.Caller;
 import com.djrapitops.plan.extension.DataExtension;
+import github.scarsz.discordsrv.DiscordSRV;
```
Comment (AuroraLS3) [inline src/main/java/com/djrapitops/extension/DiscordSRVExtensionFactory.java]: Do these imports cause a NoClassDefFoundError when discordsrv is not installed?

**Review by AuroraLS3 — COMMENTED** [review]
Looks good.

Have you tested the extension without discordsrv installed on the server?

File: src/main/java/com/djrapitops/extension/DiscordSRVExtensionFactory.java (hunk 1 above)
Comment (Vankka) [reply src/main/java/com/djrapitops/extension/DiscordSRVExtensionFactory.java]: Changed to fully qualified class names just to be sure

**Pull request merged**